installation/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ volumes:
 
 services:
   db:
-    image: mysql:5.7.22
+    image: mysql:5.7.31
```

Comment:
@EricSmekens is there a specific reason for 5.7.31?  We have no tested this version thoroughly, but would love to hear if there's a specific need on your end for this version, or you merely picked the latest available

Reply:
I believe we have been using 5.7.27 for a while, but we will soon (coming weeks) start thorough testing with 5.7.31.
I will send a update here if we've completed the testing part.

Reply:
Got it.  We've tested 5.7.26 and are comfortable updating the scripts to support this version.  Will gladly accept this PR if you can help update the script to .26 for now while we work to test .31!

Reply:
Wil do that, makes perfect sense.

Next time, I'll update with a higher version, if we tested with that thoroughly.

Reply:
*Updated